src/utils/middleware/csrf.middleware.js

```diff
@@ -0,0 +1,79 @@
+const csrf = require('csrf')
+
+/**
+ * @param logger {Logger}
+ * @param sessionKey {string}
+ * @param secretKey {string}
+ * @param tokenKey {string}
+ */
+const configureCsrfMiddleware = (logger, sessionKey, secretKey = 'csrfSecret', tokenKey = 'csrfToken') => {
```

Comment:
I might favour these parameters being something `sessionName`, `secretName`, `tokenName` just to avoid ambiguity, since e.g. a secret key could be something very different.

Reply:
yes, I think that is a sensible suggestion